aur/rkt/PKGBUILD

```diff
@@ -1,25 +1,26 @@
 # Maintainer: Yuval Adam <yuval at y3xz dot com> PGP-Key: 271386AA2EB7672F
 # Contributor: Kenny Rasschaert <kenny dot rasschaert at gmail dot com> PGP-Key: 1F70454121E41419
 
-pkgname=rocket
+pkgname=rkt
 pkgver=0.5.3
 pkgrel=1
 pkgdesc="App container runtime"
 arch=('x86_64')
-url="https://github.com/coreos/rocket"
+url="https://github.com/coreos/rkt"
 license=(apache)
 makedepends=('cpio' 'go' 'squashfs-tools')
-provides=('rocket')
-source=("https://github.com/coreos/rocket/archive/v${pkgver}.tar.gz")
+provides=('rkt')
+replaces=('rocket')
```

Comment:
Replacing rocket with rkt gives me this error though:
```
error: failed to commit transaction (conflicting files)
rkt: /usr/bin/rkt exists in filesystem
rkt: /usr/share/rkt/stage1.aci exists in filesystem
Errors occurred, no packages were upgraded.
```
I was hoping this "replaces" statement would take care of that, but it appears it doesn't do the trick.

Any suggestions?